common/src/main/scala/explore/components/Tile.scala

```diff
@@ -77,7 +77,7 @@ object Tile {
       .withHooks[Props]
       // info ref
       .useRefToVdom[html.Element]
-      .renderWithReuse { (p, infoRef) =>
```

Comment:
Well, the ref to the title area was called `info` to differentiate from the actual tile contents. Maybe `titleRef`?

Reply:
Ah ok. I thought it was the actual content. I'll revert